### ISSUE TITLE
Error when launching headless mode

### ISSUE THREAD

**Issue by PockySweet** [LeagueOfPoro/EsportsCapsuleFarmer#81]
```cmd
INFO: 2022/10/10 15:11:07 - Moving to login page
INFO: 2022/10/10 15:11:08 - Logging in
DEBUG: 2022/10/10 15:11:09 - Credentials submitted
Traceback (most recent call last):
  File "main.py", line 57, in <module>
  File "EsportsCapsuleFarmer\Setup\LoginHandler.py", line 36, in automaticLogIn
TypeError: LoginHandler.insertTwoFactorCode() takes 1 positional argument but 2 were given
[19504] Failed to execute script 'main' due to unhandled exception!
```

**Comment by xMatador-Git** [comment]
do you use 2FA?
can't reproduce it on my end
but it looks like the same error from PR #82 
changing LoginHandler.py at line 41 to 
`def insertTwoFactorCode(self):` to `def insertTwoFactorCode(self, driver):`
could be a fix

**Comment by LeagueOfPoro** [comment]
Fixed in #86
